de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/writer/WriterNodeModel.java

```diff
@@ -119,33 +120,39 @@
 
   static ScriptHandler scriptHandler;
 
-  private final WriterNodeSettings nodeSettings = new WriterNodeSettings();
+  static final String CFG_FILE = "file";
+  private final SettingsModelString nodeSettings = new SettingsModelString(CFG_FILE, null);
```

Comment:
Same.  We need to rename nodeSettings to filePath as this no longer describes a set of settings but a single setting (path).